apacheconfig/lexer.py

```diff
@@ -156,19 +156,23 @@ def t_CLOSE_TAG(self, t):
     def t_OPEN_CLOSE_TAG(self, t):
         r'<[^\n\r/]*?[^\n\r/ ]/>'
         t.value = t.value[1:-2]
-        return t
+        return self._lex_option(t)
 
     def t_OPEN_TAG(self, t):
         r'<[^\n\r]+>'
         t.value = t.value[1:-1]
-        return t
+        return self._lex_option(t)
 
     @staticmethod
     def _parse_option_value(token, lineno):
-        if not re.match(r'.*?(?:\s|=|\\\s)+', token):
+        # Grabs the first token before the first non-quoted whitespace.
+        match = re.search(r'[^=\s"\']+|"([^"]*)"|\'([^\']*)\'', token)
```

Comment:
Could `match` ever become `None`? E.g. if `token` is `=`?

Reply:
Oh, great catch!

I looked into it-- `token` should never be `=` since the `OPTION_AND_VALUE` lexing regex doesn't match the character `=`. As such it currently throws the `Illegal character` ApacheConfigError, which I think is a reasonable response.

That said, there may be more edge cases we missed! So I added a check.